### PR TITLE
Updated apiVersions and fixed doc

### PR DESCRIPTION
**Pull request by yuchaoran2011**

### CODE REVIEW

**Comment by yuchaoran2011** [comment]
@foxish @ liyinan926 Some minor updates that I thought about while working on another chart. Please take a look. Thanks!

**Comment by yuchaoran2011** [comment]
@liyinan926 Thanks for the quick review. But apparently adding your Github username in `OWNERS` didn't give you the needed permission. It didn't work for me either.

**Comment by liyinan926** [comment]
@yuchaoran2011 at least it does not grant permission to change assignee. I cannot give lgtm unless I'm the asignee.

File: incubator/sparkoperator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: sparkoperator
 description: A Helm chart for Spark on Kubernetes operator
-version: 0.1.1
+version: 0.1.2
 appVersion: latest
```
Comment (davidkarlsen) [inline incubator/sparkoperator/Chart.yaml]:
```suggestion
appVersion: v2.3.1-v1alpha1-latest
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by yuchaoran2011** [comment]
@liyinan926 I saw that you cut a release `v1alpha1-0.4-2.3.x` yesterday. So I updated `appVersion` to use that. @davidkarlsen Your comment is now addressed. Thanks!

**Comment by liyinan926** [comment]
@yuchaoran2011 Thanks! The image `gcr.io/spark-operator/spark-operator:v2.3.1-v1alpha1-latest` has been updated.

**Pull request merged**